apidoc/v2/openapi.yaml

```diff
@@ -3822,8 +3856,9 @@ paths:
         description: |
           Number of extra fields keys that will be returned. Value >= -1; -1: no limit, 0: users default setting from UCP.
         schema:
-          type: int
+          type: integer
```

Comment:
I wondered why the swagger.io schema validator (but not the less strict viewer) was complaining.

Reply:
:+1: